cmake/externalFunc.cmake

```diff
@@ -51,12 +41,12 @@ macro(addExternalFolder NAME FOLDER )
         string(TOLOWER ${CMAKE_GENERATOR} generator_lower)
         if (MSVC AND NOT ${generator_lower} STREQUAL "ninja")
             set(RadiumExternalMakeTarget ALL_BUILD)
-            message(INFO "[addExternalFolder] Enable compatibility mode for VS Generator")
+            message(VERBOSE "[addExternalFolder] Enable compatibility mode for VS Generator")
```

Comment:
I found  it's a good idea to differentiate messages emit by the build chain. For now, except those few `VERBOSE`, all messages are set to `STATUS` in the various cmake files we used. It could be helpful to ensure this PR is efficient to define a clean policy about the messages mode and to express this policy in the developer doc along Radium programming guidelines.

Reply:
Out of the scope.
This PR fixes cmake in order to remove the  errors that make the CI broken on master and radium-release. If we want a message policy we can do in another PR. 
We must stop adding more and more content to any PR submitted to Radium.

Reply:
this PR introduce a way to choose which level of message log to display, then allowing to shorten the log on the CI, then allowing the CI to success instead of stopping with "log too long" message. So, we might have coherent message mode to allow user to choose the log level they want instead of either printing everything (because every message is STATUS except those few) or nothing but the errors.